src/plugins/chatview/index.js

```diff
@@ -51,7 +51,7 @@ converse.plugins.add('converse-chatview', {
             'time_format': 'HH:mm',
             'use_system_emojis': true,
             'visible_toolbar_buttons': {
-                'call': false,
+                'call': true,
```

Comment:
This needs to be set back to `false`. This button is not related to Jingle. I'll probably just remove this button soon, but for now you can ignore it.

Reply:
>I'll probably just remove this button soon

Could you please keep this button. It has always been part of converse to enable click-to-call integration with external telephone/conferencing systems. I have one or two plugins that will break if this button is removed.

Reply:
@deleolajide: There is now an easy way via hooks to add buttons to the toolbar, so I don't really think this button should be in core.

We can move the code to a community plugin, then you can include that package and continue using it as before.